package.json

```
@@ -84,7 +84,7 @@
   },
   "devDependencies": {
     "monaco-editor": "0.19.3",
-    "pxt-blockly": "3.0.21",
+    "pxt-blockly": "3.0.23",
```

```suggestion
    "pxt-blockly": "3.0.24",
```

to pick up https://github.com/microsoft/pxt-blockly/pull/326 as well?